components/common/Buttons.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { ButtonProps, Pressable, StyleSheet } from "react-native"
+import { Headline } from "@components/Text"
+
+const darkColor = "#26282A"
```

Comment:
Should we maybe have a colors namespace (since this should be the text color as well, I guess I forgot to apply it in that PR)